module/os/linux/zfs/vdev_disk.c

```diff
@@ -1042,3 +1042,6 @@ param_set_max_auto_ashift(const char *buf, zfs_kernel_param_t *kp)
 
 	return (0);
 }
+
+ZFS_MODULE_PARAM(zfs_vdev, zfs_vdev_open_, timeout_ms, U64, ZMOD_RW,
```

Comment:
Why it is not "ZFS_MODULE_PARAM(zfs_vdev, zfs_vdev_, open_timeout_ms"?

And why do you need 64bits here?

Reply:
> Why it is not "ZFS_MODULE_PARAM(zfs_vdev, zfs_vdev_, open_timeout_ms"?
Applied.

> And why do you need 64bits here?
We probably don't need 64-bits ever for this so I made it a uint_t which is probably 32-bits in most platforms. 16-bits would be too small. I also didn't want to the type of a variable `unsigned` as it was before because it can be confusing.

Marking this issue as resolved. Feel free to re-open if you have more thoughts or feedback on this.